pkg/statestore/leveldb/migration_test.go

```diff
@@ -0,0 +1,297 @@
+// Copyright 2019 The Swarm Authors
```

Comment:
the copyright headers here are all different than what we have in bee. also wrong license.